paella-matterhorn/javascript/05_initdelegate.js

```diff
@@ -58,22 +55,25 @@ function initPaellaMatterhorn(episodeId, onSuccess, onError) {
 					       paella.matterhorn.resourceId = "";
 					   }
 					   // end #DCE logging helper
-						
-						if (serie != undefined) {
-							asyncLoader.addCallback(new paella.JSONCallback({url:'/series/'+serie+'.json'}), "serie");
-							asyncLoader.addCallback(new paella.JSONCallback({url:'/series/'+serie+'/acl.json'}), "acl");
-						
-							asyncLoader.load(function() {
-									//Check for series
-									paella.matterhorn.serie = asyncLoader.getCallback("serie").data;
-									//Check for acl
-									paella.matterhorn.acl = asyncLoader.getCallback("acl").data;
-									if (onSuccess) onSuccess();
-								},
-								function() {
-									if (onError) onError();
-								}
-							);
+					   // start #DCE get series from serch endpoint (instead of series endpoint)
+					   if (serie != undefined) {
+					       base.ajax.get({
+					           url: '/search/series.json',
+					           params: {'id': serie }
+					           },
+					           function (data, contentType, code) {
+					           var jsonData = data;
+					           if (typeof (jsonData) == "string") jsonData = JSON.parse(jsonData);
```

Comment:
A few questions:
- What happens if `jsonData` is not a string?
- Can it then be assumed to be an object?
- Does this actually happen?